src/modules/assessment/models/assessment.model.ts

```diff
@@ -63,6 +65,19 @@ export class Assessment extends BaseEntity {
     @DeleteDateColumn()
     deletedAt?: Date;
 
+    @Field(() => Boolean)
+    @Column({ type: 'boolean', default: true })
+    isActive: boolean
+
+    @FilterableField(() => String, { nullable: true })
```

Comment:
Since the `uuid` token here is used as an authorization token, exposing it here in graphql schema will leak a sensitive authorization token, to users who might not have permission to fill in the assessment.

I would advise omitting this field from being in the graphql schema and having either:
a) a @ResolveField method in an assessment resolver, which will check for necessary permissions and return the field, or null if permission is not granted.
b) a custom query to retrieve this by assessment id and check for necessary permission at that level

Option (a) would be more seamless for the frontend to consume, as it would still be presented as a field in the Assessment object, but with benefit of additional authorization check at field level